Cargo.toml

```diff
@@ -1,265 +1,16 @@
-[package]
-name = "winit"
-version = "0.29.10"
-authors = ["The winit contributors", "Pierre Krieger <pierre.krieger1708@gmail.com>"]
-description = "Cross-platform window creation library."
-edition = "2021"
-keywords = ["windowing"]
-license = "Apache-2.0"
-readme = "README.md"
-repository = "https://github.com/rust-windowing/winit"
-documentation = "https://docs.rs/winit"
-categories = ["gui"]
-rust-version = "1.70.0"
-
-[package.metadata.docs.rs]
-features = [
-    "rwh_04",
-    "rwh_05",
-    "rwh_06",
-    "serde",
-    "mint",
-    # Enabled to get docs to compile
-    "android-native-activity",
-]
-default-target = "x86_64-unknown-linux-gnu"
-# These are all tested in CI
-targets = [
-    # Windows
-    "i686-pc-windows-msvc",
-    "x86_64-pc-windows-msvc",
-    # macOS
-    "x86_64-apple-darwin",
-    # Unix (X11 & Wayland)
-    "i686-unknown-linux-gnu",
-    "x86_64-unknown-linux-gnu",
-    # iOS
-    "x86_64-apple-ios",
-    # Android
-    "aarch64-linux-android",
-    # Web
-    "wasm32-unknown-unknown",
+[workspace]
+members = [
+    "run-wasm",
+    "winit",
```

Comment:
I think it's a mistake to move `winit` to a subdirectory, at least yet, and it also complicates this PR unnecessarily. There is especially value in having the top-level `examples` directory. Let's use the [`include`](https://doc.rust-lang.org/cargo/reference/manifest.html#the-exclude-and-include-fields) field instead to specify which files should be included in the `winit `package.